src/components/Logo/index.tsx

```diff
@@ -1,22 +1,36 @@
 import { useState } from 'react'
-import { Slash } from 'react-feather'
 import { ImageProps } from 'rebass'
-import { useTheme } from 'styled-components/macro'
+import styled from 'styled-components/macro'
 
 const BAD_SRCS: { [tokenAddress: string]: true } = {}
 
 interface LogoProps extends Pick<ImageProps, 'style' | 'alt' | 'className'> {
   srcs: string[]
+  symbol?: string
+  size?: string
 }
 
+const getFontSize = (size?: string) =>
+  size ? Math.floor(parseInt(size.replace('px', '')) / 3).toString() + 'px' : '12px'
+
+const MissingImageLogo = styled.div<{ size?: string; getFontSize: (size?: string) => string }>`
+  border-radius: 100px;
+  color: ${({ theme }) => theme.textPrimary};
+  background-color: ${({ theme }) => theme.backgroundInteractive};
+  font-size: ${({ getFontSize, size }) => getFontSize(size)};
```

Comment:
Instead of passing a JS function can you use CSS `calc` so there's less logic being passed around?

Reply:
If you can't use `floor` in CSS, can you instead pass `fontSize` directly (you may not even need to go through `styled` in that case)? Calculating it in the CSS is confusing, as you're splitting where the logic executes.

Reply:
done, used calc divide by 3 as an approx, looks the same visually.